src/skip-to-content/classes.scss

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import './index';
+
+.iui-skip-to-content {
```

Comment:
Should we call it `iui-skip-to-content-button` maybe? Cause thats how it visually looks like. 
Not sure it is ok to call the component by the name of what it does versus what it is.

Reply:
Technically it's not a button, but it does visually look like a button.  I don't know, I don't have a strong feeling for or against the current name.

Reply:
I changed it to skip-to-content-link in some places. I think it's a link more so than a button, even though it has more styling than links normally do.